src/main/java/duke/DukeException.java

```diff
@@ -1,5 +1,13 @@
 package main.java.duke;
 
+/**
+ * A self-made Exception class that is supposed to handle Exceptions
+ * Not used
```

Comment:
Might want to consider using this exception to handle cases where the user input is invalid (i.e. is not a command/ inputs a task in the wrong format/ is some gibberish) -- with a custom string message